.ci/jenkins/Jenkinsfile.prod.nightly

```diff
@@ -116,8 +118,17 @@ pipeline {
                             // extract git infos
                             util.storeGitInformation(projectName)
                             env.ALREADY_BUILT_PROJECTS = "${env.ALREADY_BUILT_PROJECTS ?: ''}${projectName};"
+
+                            env["VERSION_${f}"] = maven.mvnGetVersionProperty('project.version')
                         }
                     }
+
```

Comment:
@lampajr just out of curiosity, do you really need to extract the quarkus version from the build/optaplanner-build-parent/pom.xml? Can't you extract it from buid-config.yaml the same way you did for `quarkusVersionCommunity`?

Reply:
@rgdoliveira yeah it was an alternative but with this approach we are more confident to retrieve what is actually used, whereas with the other approach we would have extracted what we set in that variable.. with no assurance that it won't be overwritten later in the build.

Reply:
@lampajr I see your point.

I just thought that it could be more confusing extracting it from the pom instead of reading it form the build-config.

I mean, the quarkus community version you extract from build-config.  The prod version is also set in the build-config (we hardcode because we know we want that version).
So for those 2 variables (`quarkusVersionCommunity` and `quarkusVersion`) we are kind of sure that we want to use what we define in the build-config.

But yeah, both ways would work.

Reply:
@Ginxo wdyt?

Reply:
well hard to decide, @lampajr's approach is more maintainable but @rgdoliveira's is clearer... 
I would suggest to use @rgdoliveira's one

Reply:
@rgdoliveira @Ginxo thanks guys, applied your suggestion here https://github.com/kiegroup/optaplanner/pull/2738/commits/a1dcfbfec34b71c8aef75c88d1f70dc6625616e4